src/containers/Dashboard.js

```diff
@@ -13,12 +13,13 @@ import {
 } from '../selectors';
 
 function mapStateToProps(state) {
+  const project = getCurrentProject(state);
```

Comment:
I think it’s worth making an explicit `getCurrentProjectReadme` here to minimize container knowledge of the store structure